swarm/src/handler.rs

```diff
@@ -93,10 +93,10 @@ pub use select::{ConnectionHandlerSelect, IntoConnectionHandlerSelect};
 /// When a connection is closed gracefully, the substreams used by the handler may still
 /// continue reading data until the remote closes its side of the connection.
 pub trait ConnectionHandler: Send + 'static {
-    /// Custom event that can be received from the outside.
-    type InEvent: fmt::Debug + Send + 'static;
-    /// Custom event that can be produced by the handler and that will be returned to the outside.
-    type OutEvent: fmt::Debug + Send + 'static;
+    /// `FromBehaviour` event that can be received from the outside.
```

Comment:
I think we can do better here.

```suggestion
    /// A type representing the message(s) a [`NetworkBehaviour`] can send to a [`ConnectionHandler`] via [`ToSwarm::NotifyHandler`]
```